docs/topics/wait_for.rst

```diff
@@ -0,0 +1,86 @@
+:orphan:
+.. currentmodule:: discord
+
+.. _guide_wait_for:
+
+Waiting for Events
+=============================
+A common step when creating commands or handling events involves having to wait for an action or user-input. To aid with this, the library offers a special method :meth:`Client.wait_for` to wait for an event in-line instead of having to define a separate event listener.
+What is ``wait_for``?
```

Comment:
```suggestion
A common step when creating commands or handling events involves having to wait for an action or user-input. To aid with this, the library offers a special method :meth:`Client.wait_for` to wait for an event in-line instead of having to define a separate event listener.

What is ``wait_for``?
```
A new line is necessary here for the header to be declared correctly.